framework/tlvf/test/main.cpp

```diff
@@ -171,16 +171,18 @@ int main(int argc, char *argv[])
         errors++;
     }
 
-    cmplx         = fourthTlv->create_complex_list();
-    cmplx->var1() = 0xd11d;
-    if (!fourthTlv->add_complex_list(cmplx)) {
-        LOG(ERROR) << "Failed to add complex list";
-        errors++;
-    }
-    if (fourthTlv->add_complex_list(cmplx)) {
-        LOG(ERROR) << "Could add complex list a second time";
-        errors++;
-    }
+    // TODO the complex list doesn't work at the moment if it has more than one element
```

Comment:
As I suggested in previous commit - please take this out to a function to keep main simple to read